crates/loader/src/local/mod.rs

```diff
@@ -188,6 +189,45 @@ async fn core(
             let name = format!("{}@{}", bindle_id, parcel_sha);
             ModuleSource::Buffer(bytes, name)
         }
+        config::RawModuleSource::Url(us) => {
+            let url = reqwest::Url::parse(&us.url)
+                .with_context(|| format!("Invalid source URL {} in component {}", us.url, id))?;
+            if url.scheme() != "https" {
+                anyhow::bail!(
+                    "Invalid URL scheme {} in component {}: must be HTTPS",
+                    url.scheme(),
+                    id
+                );
+            }
```

Comment:
I'm a bit ambivalent about this one, but its there :shrug: 
```suggestion
              anyhow::ensure!(
                  url.scheme() == "https",
                  "Invalid URL scheme {} in component {}: must be HTTPS",
                  url.scheme(),
                  id
              );
```